Jenkinsfile

```diff
@@ -32,6 +32,16 @@ pipeline{
 								checkout scm
 							}
 						}
+						stage ('Run unit tests for musl') {
+							steps {
+								sh "scripts/dev_cli.sh tests --unit --libc musl"
+							}
+						}
+						stage ('Run integration tests for musl') {
```

Comment:
We should run the musl integration tests on the master branch not the master node.